src/network/citrix/netscaler/snmp/mode/health.pm

```diff
@@ -119,6 +119,17 @@ Example: --warning='temperature,.,30'
 Set critical threshold for 'temperature', 'fanspeed', 'voltage'(syntax: type,regexp,threshold)
 Example: --critical='temperature,.*,40'
 
+=item B<--psu-new-oid>
+
+on psu mode by default, .1.3.6.1.4.1.5951.4.1.1.41.7.1.2 mean the following :
```

Comment:
Again, a couple of remarks:
- Adapt the POD content if the suggestion above is adopted.
- Explain the context a bit more: "The meaning of the OID .1.3.6.1.4.1.5951.4.1.1.41.7.1.2 has been changed between _version a_ and _version b_. To use the new mapping, add the _--final-name-of-the-option_."
- Always put a capital letter at the beginning of a sentence.